proto/zetachain/zetacore/pkg/chains/chains.proto

```diff
@@ -63,31 +78,37 @@ enum Vm {
 }
 
 // Consensus represents the consensus algorithm used by the chain
+// this can represent the consensus of a L1
+// this can also represent the solution of a L2
 enum Consensus {
   option (gogoproto.goproto_enum_stringer) = true;
   ethereum = 0;
   tendermint = 1;
   bitcoin = 2;
+  op_stack = 3;
 }
 
-//    Chain represents a blockchain network with its unique chain ID
-//    ChainName is the name of the chain
-//    ChainId is the unique identifier of the chain
-//    Network is the network type of the chain , this can be ZETA, ETH, BSC,
-//    BTC, POLYGON NetworkType is the network type of the chain, this can be
-//    MAINNET, TESTNET, DEVNET, PRIVNET Vm is the virtual machine type of the
-//    chain to support smart contracts, this can be EVM, NO_VM Consensus is the
-//    consensus algorithm used by the chain, this can be Tendermint, Ethereum,
-//    Bitcoin IsExternal is a boolean value to determine if the chain is
-//    external to Zeta IsHeaderSupported is a boolean value to determine if the
-//    chain supports headers
-
+// Chain represents static data about a blockchain network
+// it is identified by a unique chain ID
 message Chain {
-  ChainName chain_name = 1;
+  // ChainId is the unique identifier of the chain
   int64 chain_id = 2;
+
+  // ChainName is the name of the chain
+  ChainName chain_name = 1;
```

Comment:
Is the reason behind moving chain_id up to signify it is the unique identifier ?
Consider maintaining the numerical order , as the chainname is unique as well

Reply:
Yes, chain ID is the identifier, it should be first, I think it is ok using `1` for chain ID as the object is not saved in the chain store